python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -1129,6 +1129,16 @@ keys:
                   type: bool
                 unmodified_enable:
                   type: bool
+        sync_e:
+          type: dict
+          keys:
+            enable:
+              type: bool
+            priority:
+              type: str
+              description: The priority is used to influence the reference clock selection. The EOS default priority is 127. The priority can be configured to any integer between 1-255, or set to `disabled`.
```

Comment:
@ClausHolbechArista , Can we use pattern here? 
pattern: "^1-9$|^[1-9][0-9]$|^1[0-9][0-9]$|^2[0-4][0-9]$|^2[0-5][0-5]$|^disabled$"

Reply:
We could, but it is a lot of extra burden for very little benefit. We will solve this with more schema options in the future. Let's not overdo it. We have these cases in many places, and complex regex is also hard for the user.

Reply:
okay.